tests/sharding/fixtures.py

```diff
@@ -66,6 +66,8 @@ def vmc():
     )
     provider = EthereumTesterProvider(eth_tester)
     w3 = Web3(provider)
+    if hasattr(w3.eth, "enable_unaudited_features"):
+        w3.eth.enable_unaudited_features()
```

Comment:
I'll probably leave this method in, so you don't *have* to do an attribute test here. After the audit is complete it would be a noop. But no harm to check the attribute. v5 forward compatibility :P